caluma/form/serializers.py

```diff
@@ -7,3 +8,26 @@ class FormSerializer(serializers.ModelSerializer):
     class Meta:
         model = models.Form
         fields = "__all__"
+
+
+class QuestionSerializer(serializers.ModelSerializer):
+    is_required = serializers.CharField()
+    is_hidden = serializers.CharField()
+
+    def validate_is_required(self, value):
+        jexl = JEXL()
+        # TODO: define transforms e.g. answer
+        errors = list(jexl.validate(value))
+        if errors:
+            raise exceptions.ValidationError(errors)
+
+        return value
+
+    def validate_is_hidden(self, value):
+        return self.validate_is_required(value)
```

Comment:
Maybe a `_validate_jexl` method which is then called by `validate_is_required` and `validate_is_hidden` would feel a bit cleaner, don't you think?

Reply:
Fair enough. Changed it.